guessing_game_v2.html

```diff
@@ -31,92 +30,68 @@
     }
     oneOrZero();
 
-    var createName = function() {
       if (arrQuestion1.toString() === arr.toString()){
         console.log('Yaa it worked1!');
-        name.push('sas');
+        syllable.push('sas');
       } else {
         console.log("wrong1");
       }
 
       if (arrQuestion2.toString() === arr.toString()){
         console.log('Yaa it worked2!');
-        name.push('bob');
+        syllable.push('bob');
       } else {
         console.log("wrong2");
       }
 
       if (arrQuestion3.toString() === arr.toString()){
         console.log('Yaa it worked3!');
-        name.push('jue');
+        syllable.push('jue');
       } else {
         console.log("wrong3");
       }
 
       if (arrQuestion4.toString() === arr.toString()){
         console.log('Yaa it worked4!');
-        name.push('rap');
+        syllable.push('rap');
       } else {
         console.log("wrong4");
       }
 
       if (arrQuestion5.toString() === arr.toString()){
         console.log('Yaa it worked5!');
-        name.push('er');
+        syllable.push('er');
       } else {
         console.log("wrong5");
       }
 
       if (arrQuestion6.toString() === arr.toString()){
         console.log('Yaa it worked6!');
-        name.push('lap');
+        syllable.push('lap');
       } else {
         console.log("wrong6");
       }
 
       if (arrQuestion7.toString() === arr.toString()){
         console.log('Yaa it worked7!');
-        name.push('but');
+        syllable.push('but');
       } else {
         console.log("wrong7");
       }
 
       if (arrQuestion8.toString() === arr.toString()){
         console.log('Yaa it worked8!');
-        name.push('dic');
+        syllable.push('dic');
       } else {
         console.log("wrong8");
       }
-    }
-
+      console.log(syllable)
       var Person = function(syllable1, syllable2) {
```

Comment:
Not sure why you are declaring/creating your Constructor in this format. Instead of creating a variable do this:

function Person(syllable1, syllable2) {
  this.syllable1 = syllable;
  ......
}